src/shared/components/Header/Menu/MenuListItem/component.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+
+import { MenuList, MenuItem } from 'shared/types';
+import { ArrowIcon } from 'shared/icons';
+
+import { ArrowIconWrapper, ListWrapper } from './styles';
+
+type MenuListItem = {
+  list: MenuList;
+};
+
+const MenuListItem: React.FC<MenuListItem> = ({ list }: MenuListItem) => {
+  const [isOpenList, setIsOpenList] = useState(false);
+
+  const onToggleHandler = (): void => {
+    setIsOpenList(!isOpenList);
+  };
+
+  return (
+    <nav>
+      <strong
+        onClick={(): void => {
+          onToggleHandler();
+        }}
+      >
+        {list.heading}
+        <ArrowIconWrapper isOpenList={isOpenList}>
+          <ArrowIcon />
+        </ArrowIconWrapper>
+      </strong>
+      <ListWrapper isOpenList={isOpenList}>
+        {list.list.map((item: MenuItem, i: number) => (
+          <li key={i}>
```

Comment:
Maybe we could use library for keys https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Good idea, but I think that there will be no problem when we have API. but maybe I'm wrong

Reply:
Problems could begin if the order of items in the list change, that's why it's not the good practice to use `index` for `key`

Reply:
good call 👍  I think we can just add some unique `id` property to our list.
e.g.
```js
list: [
   { name: 'Як це працює', link: '/' , id: Math.random()},
...
]
```

Reply:
please update key={index} to key={item.id} for footer-list too.

Reply:
Done